cli/tests/conftest.py

```diff
@@ -81,6 +82,20 @@ def make_semgrepconfig_file(dir_path: Path, contents: str) -> None:
     semgrepconfig_path.write_text(contents)
 
 
+def make_settings_file(unique_path: Path) -> None:
+    Path(unique_path).write_text(
+        "anonymous_user_id: 5f52484c-3f82-4779-9353-b29bbd3193b6\n"
```

Comment:
would be good to comment why this magic string comes from. Does it correspond to an internal test user?